docs/ldml/tr35.md

```diff
@@ -4414,7 +4436,7 @@ Click on **Previous Version** in the header until you get to the desired version
 
 * * *
 
-© 2024–2024 Unicode, Inc.
+© 2024–2025 Unicode, Inc.
```

Comment:
```suggestion
© 2001–2025 Unicode, Inc.
```
This fixes a mistake introduced in v46; v45 had "Copyright © 2001–2024 Unicode, Inc. All Rights Reserved."
Otherwise good.